### ISSUE TITLE
How to get this to work with PRs from private repos?

### ISSUE THREAD

**Issue by andersonaddo** [z0al/dependent-issues#28]
Is there a way to allow this action to be able to see the issues and PRs of private repos that I have access to?
I've tried using a Personal Access Token that has these permissions:
<img width="744" alt="Screen Shot 2021-01-12 at 12 45 43 PM" src="https://user-images.githubusercontent.com/34724755/104354099-b6b54680-54d6-11eb-8edd-aab14ea34d37.png">
But this still doesn't work; the action still fails with `Error: Not Found`.
Any tips on what I'm doing wrong?

**Comment by z0al** [comment]
Hey there.

I think the permissions under `repo` should be enough. 

Assuming you correctly set the token variable in the workflow file, then maybe your issue is SSO? 

> If you want to use a PAT to access resources owned by an organization that uses SAML SSO, you must authorize the PAT. For more information, see "About authentication with SAML single sign-on" and "Authorizing a personal access token for use with SAML single sign-on."
> 
> \- [Official docs](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)

**Comment by andersonaddo** [comment]
Not sure about that, I don't think my orgo uses SAML SSO.
It's a standard Github organization.

Would you mind giving it a whirl and see if it works for you?

**Comment by andersonaddo** [comment]
Ah I found the problem! This is was GitHub design feature, not your fault. I had opened one PR for testing my GitHub actions, and at the time I opened it i was using the secrets.GITHUB_TOKEN token (which obviously wasn't enough). So I changed the token for the action and was still testing it in that same PR, but it looks like that PR had cached the env for the action at the time it was created (so it was still using the secrets.GITHUB_TOKEN).

I opened up a new PR and it works smoothly.
